.github/workflows/stale-checker.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Stale Checker'
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          days-before-stale: 30
+          stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove the status/stale label or comment or this will be closed in 5 days.'
+          stale-issue-label: 'status/stale'
+          days-before-issue-close: -1
```

Comment:
Should it be 5?

Reply:
currently we don't need to close stale issues since almost all these issues need to be solved.